plugins/progress/progress.plugin.sh

```diff
@@ -1,21 +1,16 @@
 #!/usr/bin/env bash
-############################---Description---###################################
-#                                                                              #
-# Summary       : Show a progress bar GUI on terminal platform                 #
-# Support       : destro.nnt@gmail.com                                         #
-# Created date  : Aug 12,2014                                                  #
-# Latest Modified date : Aug 13,2014                                           #
-#                                                                              #
-################################################################################
+#Description:
 
-############################---Usage---#########################################
+# Summary       : Show a progress bar GUI on terminal platform
+# Support       : destro.nnt@gmail.com
+# Created date  : Aug 12,2014
+# Last Modified : Feb, 21, 2020
+# Last Modifier : FG Robertson (https://github.com/grobertson)                 
 
-# Copy below functions (delay and progress fuctions) into your shell script directly
-# Then invoke progress function to show progress bar 
 
-# In other way, you could import source indirectly then using. Nothing different
-
-################################################################################
+#Usage:
+# source into your script.
+# progress int n(1...100) string<"Phase">
```

Comment:
Could you explain the reasoning for changing the header style? If we should follow this style for some reason, should we also update `lib/util.sh`, which has a similar header.